ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,40 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers returns a list of load balacners struct
```

Comment:
nit: typo for "balancers", and usually in the comment we just specify the usage for this function: only describe what this function does. Also can include an input example here like ["targetGroupArn=....,containerName=....", "...."] so that the others can be aware of the logic below.

Reply:
Oops, will add this next PR